deploy/kubernetes/README.adoc

```diff
@@ -481,4 +467,19 @@ uri:       67 bytes
 username:  4 bytes
 ----
 
+To test the Service Instance API, the URL can be constructed from the binding credentials;
+
+---
+export SI_URI=$(kubectl get secret bookstore-binding -n test -o json | jq -r '.data.uri' | base64 --decode)
```

Comment:
If we're going to suggest this command, we'll need to add `jq` as an additional pre-requisite at the top of the document. We shouldn't assume everyone will have that installed.